src/Controls/src/Core/Platform/Android/TabbedPageManager.cs

```diff
@@ -116,6 +116,7 @@ internal void SetElement(TabbedPage tabbedPage)
 
 			if (Element != null)
 			{
+				Element.InternalChildren.ForEach(page => TeardownPage(page as Page));
```

Comment:
Is this needed because the same TabbedPage gets passed in twice?

Reply:
When you return to the desktop through the system's return button and reopen it, the Element will be set to null through the DisconnectHandler, and then the Element will be reset again. Since the previous InternalChildren's Event has not been released, there is no doubt that the InternalChildren's event will be executed multiple times.